Makefile.shared

```diff
@@ -245,15 +245,15 @@ link_dso.darwin:
 	SHLIB_SUFFIX=.dylib; \
 	ALLSYMSFLAGS=''; \
 	NOALLSYMSFLAGS=''; \
-	SHAREDFLAGS="$(CFLAGS) `echo $(SHARED_LDFLAGS) | sed s/dynamiclib/bundle/`"; \
+	SHAREDFLAGS="`echo $(SHARED_LDFLAGS) | sed s/dynamiclib/bundle/`"; \
 	$(LINK_SO_DSO)
 link_shlib.darwin:
 	@ $(CALC_VERSIONS); \
 	SHLIB=lib$(LIBNAME); \
 	SHLIB_SUFFIX=.dylib; \
 	ALLSYMSFLAGS='-all_load'; \
 	NOALLSYMSFLAGS=''; \
-	SHAREDFLAGS="$(CFLAGS) $(SHARED_LDFLAGS)"; \
+	SHAREDFLAGS="$(SHARED_LDFLAGS)"; \
```

Comment:
Because `-emit-llvm` is not accepted at command line that is supposed to perform linking. But it's not like I try to justify the modification, as I'm not actually considering it right. On side note one probably should recognize that the whole thing is effectively [currently] limited to MacOS. Well, it might/should be possible to make it work on say Linux, but it's a stretch. Because by default Linux clang calls /usr/bin/ld and Linux /usr/bin/ld is incapable of linking llvm code by itself. In other words "saw" build is bound to fail [on out-of-box Linux], it fails complaining about apps/app_rand.o's file format being not recognized. apps/apps_rand.o is first-most object file referred in first-most command linking first-most executable. But why/how does it work on MacOS you may ask. Supposedly "system" linker is llvm-aware and knows what to do even if it finds llvm byte-code in object file on command line. "Supposedly" means that I personally haven't double-checked it. BTW, for reference, what would it take on Linux to link application? All object files named with .bc suffix...